tools/tm-bench/statistics.go

```diff
@@ -66,9 +66,10 @@ func calculateStatistics(
 		// increase number of blocks for that second
 		numBlocksPerSec[sec]++
 
+		// TODO: add tracking for tx/sec
 		// increase number of txs for that second
-		numTxsPerSec[sec] += blockMeta.Header.NumTxs
-		logger.Debug(fmt.Sprintf("%d txs at block height %d", blockMeta.Header.NumTxs, blockMeta.Header.Height))
+		numTxsPerSec[sec] += 1
```

Comment:
???

Reply:
below on the next line theres a TODO: so still need to get to this part